data/2015/other/Academic Policies & Procedures.xhtml

```diff
@@ -196,22 +197,14 @@
 <p class="basic-text">- the student has spent three consecutive semesters on probation.</p>
 <p class="header-paragraph">If a student is dismissed, the University will automatically cancel any registrations for the following semester.</p>
 <p class="header-paragraph">Decisions made regarding continued eligibility for Federal Financial Aid are made independently of the academic dismissal/reinstatement policy. Students should contact the Office of Scholarships and Financial Aid for assistance.</p>
-
 <p class="header-paragraph"><span class="basic-text-bold">NOTE:</span> Attempted credit hours include all the credit hours a student registered for at UNL and did not drop during the first two weeks of the course.</p>
 <p class="header-paragraph"><span class="header-paragraph-title">Readmission.</span> A student who has been dismissed from UNL will be denied enrollment privileges for at least two consecutive semesters (the four summer sessions count as one semester). Readmission to UNL is not automatic. A dismissed student may apply for readmission to UNL for the semester following the mandatory “stop-out” period or any subsequent semesters. Applications for readmission will be evaluated by the Office of Admissions in accordance with criteria established by each of the colleges.</p>
 <p class="header-paragraph">Please see college information for specific standards related to the college.</p>
-
 <p class="title-3">Probation</p>
+<p>Status AfterUNL Term GPANU Cumulative GPAAttempted UNL Credit HoursProbation &amp; Dismissal</p>
```

Comment:
This is the table headers row. It should not have been changed into a paragraph.